embassy-rp/src/flash.rs

```diff
@@ -659,6 +667,11 @@ mod ram_helpers {
         );
     }
 
+    #[inline(never)]
+    #[link_section = ".data.ram_func"]
+    #[cfg(feature = "rp235x")]
+    unsafe fn write_flash_inner(_addr: u32, _len: u32, _data: Option<&[u8]>, _ptrs: *const FlashFunctionPointers) {}
```

Comment:
please don't leave empty functions if unimplemented. Either add a `todo!()` or ideally make the API inaccessible for the user.

Reply:
I've left this as TODO, flash will be my next pr.